packages/loading/test/loading-asNumber.test.ts

```diff
@@ -367,7 +367,7 @@ describe('loading asNumbers', () => {
 			reducers: {},
 		}
 		type Models = { count: typeof count2 }
-		type ExtraModels = ExtraModelsFromLoading<Models & ExtraModels>
+		type ExtraModels = ExtraModelsFromLoading<Models>
```

Comment:
we must add this to documentation with a typescript section

Reply:
> we must add this to documentation with a typescript section

yeah, we should, can you initiate one and I'll make some additions if needed.